### PR TITLE
Renaming ratu to data_ocean

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: data_converter/urls.py [hunk 1]
```diff
@@ -18,6 +18,6 @@
 from django.urls import include, path
 
 urlpatterns = [
-    path('api/', include('ratu.urls')),
+    path('api/', include('data_ocean.urls')),
     path('admin/', admin.site.urls),
-]
\ No newline at end of file
+]
```
Comment (criticus) [inline data_converter/urls.py:23]: remove white space please

File: data_converter/wsgi.py [hunk 2]
```diff
@@ -13,4 +13,4 @@
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "data_converter.settings")
 
-application = get_wsgi_application()
\ No newline at end of file
+application = get_wsgi_application()
```
Comment (criticus) [inline data_converter/wsgi.py:16]: white space

File: data_ocean/admin.py [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+from data_ocean.models.ratu_models import Region, District, City, Citydistrict, Street
 
 admin.site.register(Rfop)
 admin.site.register(Region)
 admin.site.register(District)
 admin.site.register(City)
 admin.site.register(Citydistrict)
-admin.site.register(Street)
\ No newline at end of file
+admin.site.register(Street)
```
Comment (criticus) [inline data_ocean/admin.py:10]: and here

File: data_ocean/migrations/0036_squashed_2.py [hunk 4]
```diff
@@ -50,4 +50,4 @@ def add_kved_not_found (apps, schema_editor):
 
     operations = [
         migrations.RunPython(add_kved_not_found),
-    ]
\ No newline at end of file
+    ]
```
Comment (criticus) [inline data_ocean/migrations/0036_squashed_2.py:53]: white space

File: data_ocean/migrations/0038_auto_20200430_1655.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.0.9 on 2020-04-30 16:55
```
Comment (criticus) [inline data_ocean/migrations/0038_auto_20200430_1655.py:1]: can you please keep this PR for renaming only?

File: data_ocean/models/rfop_models.py [hunk 6]
```diff
@@ -9,4 +9,4 @@ class Rfop(models.Model):
     address = models.CharField(max_length=300, null=True)
 
     def __str__(self):
-                return self.fullname
\ No newline at end of file
+                return self.fullname
```
Comment (criticus) [inline data_ocean/models/rfop_models.py:12]: white space

File: data_ocean/serializers/rfop_serializers.py [hunk 7]
```diff
@@ -4,4 +4,4 @@ class RfopSerializer(serializers.Serializer):
     state = serializers.CharField(max_length=100)
     kved = serializers.CharField()
     fullname = serializers.CharField()
-    address = serializers.CharField()
\ No newline at end of file
+    address = serializers.CharField()
```
Comment (criticus) [inline data_ocean/serializers/rfop_serializers.py:7]: white space

File: data_ocean/services/koatuu_service.py [hunk 8]
```diff
@@ -44,4 +44,4 @@ def save_to_region_table(self, data, object_koatuu, region_koatuu_dict):
             if object_region_name in region_koatuu_dict:
                 region_koatuu = region_koatuu_dict[object_region_name]
                 region_koatuu.koatuu = object_koatuu[self.LEVEL_ONE]
-                region_koatuu.save(update_fields=['koatuu'])
\ No newline at end of file
+                region_koatuu.save(update_fields=['koatuu'])
```
Comment (criticus) [inline data_ocean/services/koatuu_service.py:47]: white space

File: data_ocean/services/main.py [hunk 9]
```diff
@@ -244,4 +244,4 @@ def done(self):
         """
         for model_name, objs in self._create_queues.items():
             if len(objs) > 0:
-                self._commit(apps.get_model(model_name))
\ No newline at end of file
+                self._commit(apps.get_model(model_name))
```
Comment (criticus) [inline data_ocean/services/main.py:247]: white space

File: data_ocean/services/ratu_service.py [hunk 10]
```diff
@@ -141,4 +141,4 @@ def save_to_street_table(self, record, region, district, city, citydistrict):
     print(
         'Ratu already imported. For start rewriting RATU to the DB run > RatuConverter().process()\n',
         'For clear all RATU tables run > RatuConverter().clear_db()'
-        )
\ No newline at end of file
+        )
```
Comment (criticus) [inline data_ocean/services/ratu_service.py:144]: white space

File: data_ocean/tests.py [hunk 11]
```diff
@@ -1,3 +1,3 @@
 from django.test import TestCase
 
-# Create your tests here.
\ No newline at end of file
+# Create your tests here.
```
Comment (criticus) [inline data_ocean/tests.py:3]: same here

File: data_ocean/urls.py [hunk 12]
```diff
@@ -13,4 +13,4 @@
     path('street/', StreetView.as_view()),
     path('citydistrict/', CitydistrictView.as_view()),
     path('district/', DistrictView.as_view()),
-]
\ No newline at end of file
+]
```
Comment (criticus) [inline data_ocean/urls.py:16]: and here

File: manage.py [hunk 13]
```diff
@@ -12,4 +12,5 @@
             "available on your PYTHONPATH environment variable? Did you "
             "forget to activate a virtual environment?"
         ) from exc
-    execute_from_command_line(sys.argv)
\ No newline at end of file
+    execute_from_command_line(sys.argv)
+
```
Comment (criticus) [inline manage.py:16]: too many empty lines

File: script.py [hunk 14]
```diff
@@ -11,4 +11,4 @@
     if RfopConverter().unzip_file() == 0:
         RfopConverter().rename_files()
         # RfopConverter().process()
-        # RuoConverter().process()
\ No newline at end of file
+        # RuoConverter().process()
```
Comment (criticus) [inline script.py:14]: white space

File: data_converter/urls.py (hunk 1 above)
Comment (IegorSoboliev) [reply data_converter/urls.py:23]: done

File: data_converter/wsgi.py (hunk 2 above)
Comment (IegorSoboliev) [reply data_converter/wsgi.py:16]: done

File: data_ocean/admin.py (hunk 3 above)
Comment (IegorSoboliev) [reply data_ocean/admin.py:10]: done

File: data_ocean/migrations/0036_squashed_2.py (hunk 4 above)
Comment (IegorSoboliev) [reply data_ocean/migrations/0036_squashed_2.py:53]: done

File: data_ocean/migrations/0038_auto_20200430_1655.py (hunk 5 above)
Comment (IegorSoboliev) [reply data_ocean/migrations/0038_auto_20200430_1655.py:1]: ok

Comment (criticus) [reply data_ocean/migrations/0038_auto_20200430_1655.py:1]: I still see this migration in the PR

Comment (IegorSoboliev) [reply data_ocean/migrations/0038_auto_20200430_1655.py:1]: Sorry, I have thought we need this migration for the new model Category. Should I delete it?

**Pull request merged**